sxs/utilities/lvcnr/conversion.py

```diff
@@ -165,7 +170,7 @@ def convert(self, sxs_data_path, out_path, truncation_time=None, resolution=None
         log("Output filename is '{0}'".format(out_name))
 
         start_time, peak_time, version_hist = convert_modes(
-            sxs_data_path + "/rhOverM_Asymptotic_GeometricUnits_CoM.h5",
+            sxs_data_path + "/Strain_N2",
```

Comment:
In case — e.g., Geoffrey — uses this at some point, could you add logic like the above to prefer Strain_N2, but if it doesn't exist choose rhOverM?

Reply:
I'll do this and also just make it an optional argument to provide a waveform name